toponetx/datasets/graph.py

```diff
@@ -0,0 +1,92 @@
+"""Various examples of named graphs represented as complexes."""
+
+import networkx as nx
+import numpy as np
+
+from toponetx import CellComplex
+from toponetx.algorithms.spectrum import (
+    hodge_laplacian_eigenvectors,
+    set_hodge_laplacian_eigenvector_attrs,
+)
+from toponetx.transform.graph_to_simplicial_complex import graph_2_clique_complex
+
+__all__ = ["karate_club"]
+
+
+def karate_club(complex_type="simplicial", feat_dim=2):
+    """Load the karate club as complex.
+
+        simplicial complex is returned as the clique complex of the graph
```

Comment:
Sentence needs to start with an uppercase, e.g. A simplicial complex ...
Ditto for the sentence below.